app/helpers/tasklist_helper.rb

```diff
@@ -31,9 +31,11 @@ def subsection_link(subsection, log, current_user)
 
   def review_log_text(log)
     if log.collection_period_open?
-      link = log.sales? ? review_sales_log_path(id: log, sales_log: true) : review_lettings_log_path(log)
+      path = log.sales? ? review_sales_log_path(id: log, sales_log: true) : review_lettings_log_path(log)
 
-      "You can #{govuk_link_to 'review and make changes to this log', link} until #{log.form.end_date.to_formatted_s(:govuk_date)}.".html_safe
+      date = log.collection_start_year == 2022 ? Time.zone.local(2023, 6, 9) : log.form.end_date
```

Comment:
So my preference here is that the magic dates belong in the form definitions. If we’re defining a new date to display to users, albeit a fake one, then it should also belong there. How about display_end_date or similar, with the default that `@display_end_date = @end_date`?

Reply:
I'm happy with that. I think that whatever is written in this PR will hopefully be superseded pretty soon by something a bit more lasting anyway!